### ISSUE TITLE
set brush color is not working when we have using setbrush  api method with "LINE_DRAWING" 

### ISSUE THREAD

**Issue by sunilbenur45** [nhn/tui.image-editor#409]
instance.startDrawingMode('LINE_DRAWING');
        instance.setBrush({
            width: '10',
            color: 'white'
        });

but it will work for the free drawing mode

**Comment by sardarmd4** [comment]
Hi,
I am also facing this issue. Please help

**Comment by jinwoo-kim-nhn** [comment]
Thanks. I will correct it after checking.

**Comment by sunilbenur45** [comment]
Not only color even with size also

**Comment by yennor** [comment]
I've got the same problem. for the moment the workaround I use is to call start drawing mode with the extra parameter:
```
{
      width: this.brushWidth,
      color: this.color
    }
```

and when ever one of the parameters change I call stopDrawingMode and then startDrawingMode again...

**Comment by pedubreuil** [comment]
Hi! 
I have the same issue... Any update about its fix? 
Thanks in advance!

**Comment by lja1018** [comment]
@sunilbenur45 @sardarmd4 @yennor @pedubreuil 
This issue was resolved in v3.11.0. Thank you!
